radio/src/gui/480x272/theme.h

```diff
@@ -26,7 +26,7 @@ class BitmapBuffer;
 #define MAX_THEME_OPTIONS              5
 
 class Theme;
-void registerTheme(Theme * theme);
+extern void registerTheme(Theme * theme);
```

Comment:
https://github.com/mpaperno/opentx/blob/e707d9c58e6adfd8762534a9ea98998e234329b4/radio/src/gui/480x272/menus.h#L243

Not many, though I count ~ 36  ;-)   I guess from "legacy" code as Bertrand points out.